packages/core-mobile/app/store/balance/slice.ts

```diff
@@ -7,14 +7,20 @@ import {
 import { RootState } from 'store'
 import { selectActiveAccount } from 'store/account'
 import { selectActiveNetwork, selectAllNetworks } from 'store/network'
-import { Network } from '@avalabs/core-chains-sdk'
+import { ChainId, Network } from '@avalabs/core-chains-sdk'
 import { selectIsDeveloperMode } from 'store/settings/advanced'
 import { TokenType } from '@avalabs/vm-module-types'
 import {
   isTokenWithBalanceAVM,
   isTokenWithBalancePVM
 } from '@avalabs/avalanche-module'
 import { TokenVisibility } from 'store/portfolio'
+import {
+  AssetBalanceSort,
+  AssetNetworkFilter
+} from 'new/features/portfolio/components/assets/consts'
```

Comment:
this might cause the tsc error again since you are using the new folder here. anyway, let's just move these types to balance/types since this is the source of truth.